halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/security/OAuth2.java

```diff
@@ -200,4 +213,8 @@ public static Provider fromString(String name) {
               + Arrays.toString(Provider.values()));
     }
   }
+
+  private static boolean isSet(String config) {
+    return config != null;
```

Comment:
maybe you can reuse a common library like the one from apache or from the SpringFramework (StringUtils)

Reply:
I don't think this method is necessary at all.. 

`if (isSet(x)) === if (x)`